src/parkapi_sources/converters/apcoa/mapper.py

```diff
@@ -0,0 +1,101 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from collections import defaultdict
+from datetime import datetime, timezone
+
+from parkapi_sources.models import StaticParkingSiteInput
+from parkapi_sources.models.enums import PurposeType
+
+from .validators import (
+    ApcoaNavigationLocationType,
+    ApcoaOpeningHoursTime,
+    ApcoaOpeningHoursWeekdays,
+    ApcoaParkingSiteInput,
+    ApcoaParkingSpaceType,
+)
+
+
+class ApcoaMapper:
+    @staticmethod
+    def map_static_parking_site(apcoa_input: ApcoaParkingSiteInput) -> StaticParkingSiteInput:
+        latitude, longitude = next(
+            iter(
+                (navigation_locations_input.GeoCoordinates.Latitude, navigation_locations_input.GeoCoordinates.Longitude)
+                if navigation_locations_input.LocationType == ApcoaNavigationLocationType.DEFAULT
+                else (None, None)
```

Comment:
Why this line? You This gives invalid data in case of missing `ApcoaNavigationLocationType.DEFAULT`. You should check for the existence of `ApcoaNavigationLocationType.DEFAULT`  in your validator, and here just rely on that.